htdocs/expedition/card.php

```diff
@@ -1042,6 +1042,7 @@
     	    while ($i < $numAsked)
     	    {
     	    	print 'jQuery("#qtyl'.$i.'").val(jQuery("#qtyasked'.$i.'").val() - jQuery("#qtydelivered'.$i.'").val());'."\n";
+                if (!empty($conf->productbatch->enabled)) print 'jQuery("#qtyl'.$i.'_'.$i.'").val(jQuery("#qtyasked'.$i.'").val() - jQuery("#qtydelivered'.$i.'").val());'."\n";
```

Comment:
Imagine we have a qty to ship of 10 (already shipped of 0) and the page suggest by default 3 on lot A (we have only 3 of lot A), 3 on lot B (we have only 3 on lot B), 4 on lot C (it remains only 4 on lot C)
If we add this, when we will click on "Autofill", we will get the 3 lines with qty
10 on line with lot A
10 on line with lot B
10 on line with lot C
So the autofill will fill with qty of 30.

Reply:
Yes, It's a little tricky. But I think that this first implementation  ( that is better than no possibility to autofill nor reset ) may be improved later.

Reply:
The "reset" is safe, so i think we can introduce it.
And use a hidden option to introduce fill. The hidden option ill be removed once the feature for autofill for serial will be stable with no risk to fill badly.

Reply:
I think this is a good option as the set option can be obtained simply by refreshing the page.